### PR TITLE
small ux fixes from feedback

### PR DESCRIPTION
**Pull request by katiedavis**
### WHAT is this pull request doing?
This fixes the broken help text on create as well as adds all the options available to users. 
It also adds green text to `webhook` and `billing` selections

After feedback from @gfscott I've updated the options help to look like this:
Screen Shot 2019-11-01 at 10 38 26 AM

!--
  Summary of the changes committed.
  Before / after screenshots appreciated for UI changes.
-->

### CODE REVIEW

File: lib/shopify-cli/commands/create.rb [hunk 1]
```diff
@@ -21,8 +21,12 @@ def self.extended_help
           {{bold:Subcommands:}}
 
             {{cyan:project}}: Creates an app based on type selected.
-              Usage: {{command:#{ShopifyCli::TOOL_NAME} create project <appname>}} or
-                     {{command:#{ShopifyCli::TOOL_NAME} create project <appname> --type=TYPE}]
+              Usage: {{command:#{ShopifyCli::TOOL_NAME} create project <appname>}} or any combination of
```
Comment (tanema) [inline lib/shopify-cli/commands/create.rb]: Do we need to repeat the command over and over? Does that send the right message? (It is possible everyone already had this discussion, sorry)

Comment (katiedavis) [reply lib/shopify-cli/commands/create.rb]: Very good question! We haven’t really discussed. It does seem redundant for sure

Comment (gfscott) [reply lib/shopify-cli/commands/create.rb]: I don't think we discussed before, but agreed, I think we can do this without repeating the command each time. How about:

```
Usage: 
  {command}

Options:
  --type      {definition}
  --title     {definition}
...etc
```

Comment (katiedavis) [reply lib/shopify-cli/commands/create.rb]: I like this way more!

**Comment by davekilljoy** [comment]
`Options` should be its own section

Screen Shot 2019-11-01 at 12 06 46 PM

File: lib/shopify-cli/app_types/node.rb [hunk 2]
```diff
@@ -95,7 +95,7 @@ def check_npm_node
           dep_name = dep.split.first
           dep_link = dep_name == 'node' ? 'https://nodejs.org/en/download.' : 'https://www.npmjs.com/get-npm'
           version, stat = ctx.capture2e(dep)
-          ctx.puts("{{green:✔︎}} #{dep_name} #{version}")
+          ctx.puts("{{v}} #{dep_name} #{version}")
```
Comment (tylerball) [inline lib/shopify-cli/app_types/node.rb]: is that a `v` or a checkmark?

Comment (katiedavis) [reply lib/shopify-cli/app_types/node.rb]: that is a v! 
that's what the ui kit uses for checkmarks
`CHECK    = new('v', 0x2713,  Color::GREEN)`

Comment (tylerball) [reply lib/shopify-cli/app_types/node.rb]: oh, sweeeet

**Pull request merged**